wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -205,11 +205,15 @@ instance ToDocs Account where
   readOnlyFields   = readOnlyFieldsFromJSON
   descriptionFor _ = "An Account."
 
+instance ToDocs WalletAddress where
+  readOnlyFields   = readOnlyFieldsFromJSON
+  descriptionFor _ = "An Address."
+
 instance ToDocs AccountUpdate where
   descriptionFor _ = updateDescr (Proxy @Account)
 
 instance ToDocs Address where
-  descriptionFor _ = "An Address."
+  descriptionFor _ = "An Address ID."
```

Comment:
This is really a "Cardano Address". Let's call things the way they should be called 😁  Also, document the format, for example that they are in base58.